### PR TITLE
add breakline at the end of function

### PR DESCRIPTION
**Pull request by lykimq**
Add breakline at the end of the function. It helps to read the codes easier. 

Note from @d4hines:
- Don't squash this PR when merging
- To rebase your command, see the commit message of the second commit:

```
chore: reformat the code with new module spacing

Instructions for rebasing on this PR:

Try a normal `git rebase`.

If that doesn't work, do the following:
- Rebase on the commit previous to this one: 349b7e8c577e5f71a08995a6b298f2cce41ba1b6. This will change only .ocamlformat.
- Then do this command
  `git rebase -i --exec "nix develop -c dune build @fmt --auto" origin/main`
- You'll be prompted to ammend each commit that needs changes.
```

### CODE REVIEW

**Comment by aguillon** [comment]
As I said in the previous PR: I'm not against this, but the line breaks were here for a reason, and other people could disapprove this PR. Please comment.

File: .ocamlformat [hunk 1]
```diff
@@ -23,4 +23,5 @@ if-then-else = fit-or-vertical
 let-and = sparse
 type-decl = sparse
 sequence-blank-line=preserve-one
-module-item-spacing=preserve
+#module-item-spacing=preserve
```
Comment (d4hines) [inline .ocamlformat:26]:
```suggestion
```

**Review by d4hines — COMMENTED** [review]
Please remove the config option rather than comment out. 

Besides that I don't have a preference on this change.

**Comment by aguillon** [comment]
Ok, it seems like we're going to adopt this. I believe (perhaps wrongly) that it's going to be annoying to rebase onto. Any advice on how we should do the transition?

File: .ocamlformat [hunk 2]
```diff
@@ -22,5 +22,5 @@ nested-match = align
 if-then-else = fit-or-vertical
 let-and = sparse
 type-decl = sparse
-sequence-blank-line=preserve-one
```
[the remark this answers is not in the document]
Comment (lykimq) [reply .ocamlformat:25]: > This isn't actually a change, is it?

Yes, it is, but it was duplicated (which the same at the end) that why I remove it at this line.

**Review by d4hines — APPROVED** [review]
LGTM

**Pull request merged**